quizzes/schema.py

```diff
@@ -70,18 +89,55 @@ def resolve_classes(self, info, **kwargs):
             '''
             grabs every class that contains a teacher with this email address
 
-            << tableName >>__<< field >>__<< contains >> = << value to search for >>
+            << tableName >>__<< field >>__contains = << value to search for >>
             NOTE: these are double underscores between each field
             '''
-            return Class.objects.filter(
-                TeacherID__TeacherEmail__contains=teacher.TeacherEmail
-                )
+
+            return teacher.class_set.all()
+            
+            # return Class.objects.filter(
+            #     TeacherID__TeacherEmail__contains=teacher.TeacherEmail
+            #     )
 
         except:
             raise GraphQLError('Please supply a valid JWT')
```

Comment:
For any GraphQLErrors, these can be displayed to the end user, so it may be beneficial to make them more generic. For example, a user likely won't know what a JWT is, however they will understand needing to log in.

Reply:
The user shouldn't see this message. This specific portion is for returning a Teachers Classrooms. If this error message appears then something is wrong with the user: JWT has been tampered with, wrong ID saved in the JWT, etc.

Reply:
Not to mention this query will probably be removed in favor of the "global" `teacher` query this PR introduces. Most likely, next week will be cleaning up unused code.

Reply:
Fair enough. My thought was that anything which is carried on `error` can be something we display to the user. If we're going to clear it out completely later anyway then there's no reason to bother changing this one at all.